.travis.yml

```diff
@@ -56,14 +56,13 @@ script:
   - export TEST_EXTRA=""
   - if [[ "$COVERAGE" == "true" ]]; then
       export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    fi
-  - if [[ "$STATIC" == "false" ]]; then
+    elif [[ "$STATIC" == "true" ]]; then
```

Comment:
> I think it's partly because the 'if' is assumed to be true for branch prediction way way under the hood, which is probably not relevant here

I think modern branch predictors are likely so advanced that this makes no difference. They might assume true the very first time, but after one false that's all out the window.

I just hate scrolling through a huge if condition to get to the smaller conditions. Again, it doesn't matter here since they're all quite small, but if you had HUGE->SMALL->SMALL you'd never get the starts of all of them on the screen at once, whereas if you have SMALL->SMALL->HUGE you can see the start of each condition on one screen, giving you a sense of what the condition is and what it's doing. I read it in a programming book one time, it's a thing :)